openquake/calculators/postproc/plots.py

```diff
@@ -229,7 +229,10 @@ def plot_rupture(rup, backend=None, figsize=(10, 10),
         import matplotlib
         matplotlib.use(backend)
     _fig, ax = plt.subplots(figsize=figsize)
-    ax.set_title(f"length={rup.surface.length}, width={rup.surface.width}")
+    if hasattr(rup.surface, 'length') and hasattr(rup.surface, 'width'):
+        ax.set_title(f"length={rup.surface.length}, width={rup.surface.width}")
+    elif hasattr(rup.surface, 'get_width') and hasattr(rup.surface, 'get_area'):
+        ax.set_title(f"width={rup.surface.get_width()}, area={rup.surface.get_area()}")
```

Comment:
No good. The methods that exists for all surfaces are `get_area` and `get_width`, so lets use only those.